### DISCUSSION TITLE
Add extra data to order

### DISCUSSION THREAD

**Discussion by freshface** [duncanmcclean/simple-commerce#715] [Help] [answered]
Is it possible to add extra data to an order? To link the order to an entry from another collection?

**Comment by duncanmcclean** [comment, marked as answer]
Hello 👋

Yes, this is possible. Just add the field to the order blueprint and you're good to go.

If you want to be able to update that field in your front-end, add the field to the [field whitelist](https://simple-commerce.duncanmcclean.com/tags#content-field-whitelisting).
